internal/generator/edwards/template/point.go.tmpl

```diff
@@ -496,7 +502,6 @@ func (p *PointExtended) Add(p1, p2 *PointExtended) *PointExtended {
 // MixedAdd adds a point in extended coordinates to a point in affine coordinates
 // https://hyperelliptic.org/EFD/g1p/auto-twisted-extended-1.html#addition-madd-2008-hwcd-4
```

Comment:
change to: https://hyperelliptic.org/EFD/g1p/auto-twisted-extended.html#addition-madd-2008-hwcd-2